dotnet/src/SemanticKernel.Core/Kernel.cs

```diff
@@ -41,11 +41,6 @@ public sealed class Kernel : IKernel, IDisposable
     /// <inheritdoc/>
     public IPromptTemplateEngine PromptTemplateEngine { get; }
 
-    /// <summary>
-    /// Return a new instance of the kernel builder, used to build and configure kernel instances.
-    /// </summary>
-    public static KernelBuilder Builder => new();
```

Comment:
This is definitely breaking change. I would propose to keep all changes in this PR as it is but mark this field as `Obsolete` for some period of time, so users will be able to migrate to another syntax smoothly. 
```csharp
/// <summary>
/// Return a new instance of the kernel builder, used to build and configure kernel instances.
/// </summary>
[Obsolete("This field will be removed in a future release. Initialize KernelBuilder through constructor instead (new KernelBuilder()).")]
public static KernelBuilder Builder => new();
```